server/community.go

```diff
@@ -32,3 +33,19 @@ func (s *Server) addHacktoberfestLabel(ctx context.Context, pr *model.PullReques
 		return
 	}
 }
+
+func (s *Server) PostPRWelcomeMessage(ctx context.Context, pr *model.PullRequest) error {
```

Comment:
Any reason this is exported now? The test file is in the same package, so it will still be able to access if it's not exported.

Reply:
@agnivade I was thinking about moving the test into a separate package. Does this make sense to you? If not, yes, then there is no reason to export it.

Reply:
Having a separate test package is great, but if we have to export methods _just_ for that purpose, then we can keep it in the same package. It's more important to remain consistent IMO.